galeracluster/source/documentation/mysql-wsrep-options.rst

```diff
@@ -1141,7 +1141,7 @@ Defines the Online Schema Upgrade method the node uses to replicate :abbr:`DDL (
    "Dynamic Variable", "Yes"
    "Permitted Values", "Enumeration"
    "Default Value", "``TOI``"
-   "Valid Values", "``TOI``, ``RSU``"
+   "Valid Values", "``TOI``, ``RSU``", ``NBO``"
    "Initial Version", "MySQL-wsrep: 5.5.17-22.3, MariaDB: 5.5.21"
```

Comment:
NBO is in MariaDB Enterprise only, we need a better way to say initial version here (and to be fair, there is a version in Percona XtraDB Cluster (PXC))